html/elements/input/datetime-local.json

```diff
@@ -20,7 +20,17 @@
               },
               "firefox": {
                 "version_added": "93",
-                "notes": "Before Firefox 144, only displays a date picker without a time picker, see [bug 1726108](https://bugzil.la/1726108)."
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "dom.forms.datetime.timepicker",
+                    "value_to_set": "true"
+                  }
+                ],
+                "notes": [
+                  "Before Firefox 144, only displays a date picker without a time picker, see [bug 1726108](https://bugzil.la/1726108).",
+                  "There is an issue when a `min` and/or `max` attribute is set with a time value the time picker has no values, see [bug 1990226](https://bugzil.la/1990226)."
+                ]
```

Comment:
The flag is irrelevant here, because Firefox does support this feature, providing a user interface to modify both date and time. Like Safari, Firefox doesn't provide a time picker, but this is not explicitly required in the spec.

We should revert the notes to mainly state the same as Safari, mentioning [bug 1726107](https://bugzilla.mozilla.org/show_bug.cgi?id=1726107).